db/schema.rb

```diff
@@ -33,10 +33,14 @@
     t.string "name"
     t.string "email"
     t.string "position"
+<<<<<<< HEAD
```

Comment:
This shouldn't be here. These are Git anotation from an earlier rebase.

Reply:
Already removed